src/Mealz/MealBundle/Controller/ParticipantController.php

```diff
@@ -232,4 +208,19 @@ public function editParticipation(Week $week)
             'prototype' => $prototype,
         ));
     }
+
+    private function generateResponse(string $route, string $action, Participant $participant): JsonResponse
+    {
+        $response = new JsonResponse();
+        $response->setData(
```

Comment:
I believe data can be specified as the first constructor parameter to JsonResponse. We can save an extra method call this way. May be something like "return new JsonResponse([...]);".

Reply:
Done.